lib/server.js

```diff
@@ -153,6 +153,7 @@ function MorayServer(options) {
         help: 'The node-fast CRC compatibilty mode of the Fast server'
     }).set(crc_mode);
 
+    collector.FIXED_BUCKETS = true;
```

Comment:
@joshwilsdon Is this right? I can't tell from https://github.com/joyent/node-artedi/blob/master/docs/API.md#collectorfixed_buckets nor quickly from a scan of https://github.com/joyent/node-artedi/pull/17 how FIXED_BUCKETS is meant to be used.

If it is "always true in node-artedi v2", then is there a point in setting it here?

Reply:
node-fast checks for `FIXED_BUCKETS` [here](https://github.com/joyent/node-fast/blob/7678aa13604d6ae903676764f68ee3db2c52dea4/lib/fast_server.js#L246).

Reply:
@khalfella Per Josh's comment at https://github.com/joyent/moray/pull/17#pullrequestreview-341474073   I think you are not meant to *set* this. It is a value that is set by node-artedi automatically.

Reply:
I commented on this same issue in #17 but the short answer is: no. It should not be doing this.

Reply:
@joshwilsdon  - Thanks for making this clearer. I removed the line that sets `FIXED_BUCKETS` and confirmed that we still get correct histograms.